src/server/Document/component.jsx

```diff
@@ -32,6 +33,7 @@ const Document = ({
     min-height: 100vh;
     display: flex;
     flex-direction: column;
+    background-color: ${C_GHOST};
```

Comment:
In the same PR, can we please remove the usage of `C_GHOST` elsewhere in the application? e.g. in the `src/app/components/Grid/index.jsx` & `src/app/lib/styledGrid/index.jsx`.
Then replacing `GhostGelPageGrid` with `GelPageGrid` & `GhostGrid` with `Grid`. 
Thanks.

Reply:
I've done the above now ^

I just had to name Grid to GridWrapper. This is because CpsRelatedContent page is currently importing in `Grid`: https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsRelatedContent/index.jsx#L20

Reply:
Thanks Khoa, that looks good to me 👍